test-data/unit/check-classes.test

```diff
@@ -1975,7 +1975,7 @@ class B:
     a = A
     bad = lambda: 42
 
-B().bad() # E: Invalid method type
+B().bad() # E: Attribute function of type Callable[[], int] does not accept self argument
```

Comment:
I'd tweak message to be `Attribute function with type Callable[...] ...`. Otherwise this could be understood to refer to an attribute of an object, where the object is callable.